app/view_objects/missing_data_view.rb

```diff
@@ -31,7 +31,7 @@ def summary_items_content
   end
 
   def invalid_data?
-    missing_fields.flatten.size.positive?
+    false
```

Comment:
Im still not sure if this is the best thing to do as it could be used by other areas. I think we just want to change `ReviewSummary::View` to only return the `ErrorSummary` in line 18 of its file. and remove the `invalid_data_view.invalid_data?` in the `render?` method within the same file

Reply:
![image](https://user-images.githubusercontent.com/50492247/135235908-99af5c28-9cda-4a51-b500-3ab4ab916984.png)
That gets rid of this invalid data summary box, which I don't think is what we want. cc @edwardhorsford

Reply:
Ah that's right. I need to get my head around how we wired this up again, let's pair after

Reply:
Yeah the intention is to just remove for missing data.